.storybook/Select.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Box, Card, Select, SelectField } from '../src'
+
+const SelectFieldCard = Card.extend`
+  height: 200px;
+  overflow: scroll;
+`
+
+storiesOf('Select', module)
+  .add('Select Composition - Collapsed', () => (
+    <Box width={220}>
+      <Select># Rooms</Select>
+    </Box>
+  ))
+  .add('Select & SelectField Composition - Expanded', () => (
+    <Box width={220}>
+      <Select isExpanded># Rooms</Select>
+      <SelectFieldCard borderWidth={0} boxShadowSize="lg" mt={2}>
+        <SelectField isSelected>1 Room</SelectField>
+        <SelectField>2 Rooms</SelectField>
```

Comment:
So, I'm guessing this isn't the `<select>` element – otherwise these would be `<option>` elements right?

Reply:
No, the select is a completely separate component and should not be confused with this, which is a Dropdown (this is also the term used in the Sketch document) – we have separate use cases for both. For Select, we'll want a styled [`<select>`](https://html.spec.whatwg.org/#the-select-element) – see my comments below

Reply:
@jxnblk ,
Caught up with the rest of the review comments, and spoke to @hakimelek to avoid confusion also. Gotcha. Will rename the components to `Dropdown` and `DropdownField` for this PR. No más confusion between dropdown and select.

Will mention you in comments for review again, once code is updated. 🙏